xbmc/storage/MediaManager.h

```diff
@@ -87,6 +87,9 @@ class CMediaManager : public IStorageEventsCallback, public IJobCallback
   void OnStorageUnsafelyRemoved(const std::string &label) override;
 
   void OnJobComplete(unsigned int jobID, bool success, CJob *job) override { }
+  
+  bool playStubFile(const CFileItem& item);
```

Comment:
looks like `#include "xbmc/FileItem.h"` is missing for building on all platforms

Reply:
Thanks, fixed that. Not sure why it compiles fine without that include on Windows but fails on everything else.